src/pages/index.fr-fr.js

```diff
@@ -139,6 +139,9 @@ const Outro = () => (
               <li>
                 <a href="/projects">Liste de projets basés sur eBPF</a>
               </li>
+              <li>
+                <a href="https://www.ebpf.top">Tête première eBPF (Chinois)</a>
```

Comment:
Hi and thanks! This is not entirely correct in French. I don't read Chinese, but a literal translation of the English title above could be `eBPF, la tête la première`; or `Plonger la tête la première dans eBPF` (“Dive head first into eBPF”), or just `Plongée dans eBPF` (“A dive into eBPF”).

Reply:
Thanks a lot!